.env.example

```diff
@@ -11,4 +11,8 @@ PAGOPA_PORT=3002
 APPINSIGHTS_DISABLE=false
 APPINSIGHTS_SAMPLING_PERCENTAGE=100
 
-APPINSIGHTS_INSTRUMENTATIONKEY=key
\ No newline at end of file
+APPINSIGHTS_INSTRUMENTATIONKEY=key
+
+PAGOPA_WS_NODO_PER_PSP_URI=/webservices/pof/PagamentiTelematiciPspNodoservice
```

Comment:
What are this new environment variables used for?

Reply:
`io-pagopa-node-mock` exposes all webservices with:
- /webservices/pof/PagamentiTelematiciPspNodoservice

according to this [configuration](https://github.com/pagopa/io-pagopa-node-mock/blob/71e8896e421bbd7a1fb8d4fb478acf71400f91d7/src/config.ts#L28).
In this [PR](https://github.com/pagopa/io-pagopa-proxy/pull/320) default values of ws base paths have changed, so the updated `env.example` can be used locally with `io-pagopa-node-mock`.